samples/boards/stm32/pm_lptim/src/main.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2020 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+#include <string.h>
+#include <soc.h>
+#include <device.h>
+#include <drivers/gpio.h>
+#include <sys/printk.h>
+
+struct device *gpio_bn, *gpio_led;
+
+bool ready_to_sleep;
+
+/* in msec */
+/* setting to K_TICKS_FOREVER willl activate the deepsleep mode */
+#define SLEEP_TIME 3000
+
+#define DEMO_DESCRIPTION	\
+	"Demo Description\n"	\
+	"Application is demonstrating the low power" \
+	"with the LPTIM used as systick.\n" \
+	"The system goes to sleep mode on k_sleep() when pressing the button\n" \
+	"(led off) and wakes up by the lptim timeout expiration (led on)\n\n" \
+
+/* The devicetree node identifier for the "sw0" alias. */
+#define SW0_NODE DT_ALIAS(sw0)
+/* GPIO for the User push button */
+#if DT_NODE_HAS_STATUS(SW0_NODE, okay)
+#define SW	DT_GPIO_LABEL(SW0_NODE, gpios)
+#define PIN	DT_GPIO_PIN(SW0_NODE, gpios)
+#if DT_PHA_HAS_CELL(SW0_NODE, gpios, flags)
```

Comment:
You don't need to make this conditional now that #27076 has been merged.

Reply:
this part is similar to the existing `samples/basic/button/src/main.c`

Reply:
No, it's similar to `button` before the sample got fixed up.

Reply:
right, removed